start.sh

```diff
@@ -14,63 +14,73 @@ if [[ -z "$@" ]]; then
 else
 	CONTAINERS="$@"
 fi
+
 URL_basic_wordpress="http://${BASIC_HOST:-basic.wordpress.test}"
 URL_woocommerce_wordpress="http://${WOOCOMMERCE_HOST:-woocommerce.wordpress.test}"
 URL_multisite_wordpress="http://${MULTISITE_HOST:-multisite.wordpress.test}"
 URL_standalone_wordpress="http://${STANDALONE_HOST:-standalone.wordpress.test}"
 
+PORT_basic_wordpress=1987
+PORT_woocommerce_wordpress=1988
+PORT_multisite_wordpress=1989
+PORT_standalone_wordpress=1990
+
+USER_ID=`id -u`
+GROUP_ID=`id -g`
+
+cp containers/wordpress/Dockerfile.template containers/wordpress/Dockerfile
+sed -i -e "s/\$UID/$USER_ID/g" containers/wordpress/Dockerfile
+sed -i -e "s/\$GID/$GROUP_ID/g" containers/wordpress/Dockerfile
```

Comment:
I think it's good practice to use curly brackets for variables as much as possible: `${USER_ID}/g`. This prevents trailing text to be included in the variable name.